### ISSUE TITLE
[Data Explorer] Meta tracking issue

### ISSUE THREAD

**Issue by ashwin-pc** [opensearch-project/OpenSearch-Dashboards#4216]
### Timeline 

> Numbers in brackets are to indicate complexities

#### 2.9: Data Explorer with Discover 2.0 experimental 

1.  Week 1 - 2 (June)
    - Create `data_explorer` branch and associated plugin structure. (1)
    - Begin development of the basic 'data_explorer' plugin structure. (3)
    - [Research] switching functionality between 'discover' and 'discover_legacy'. (2)
    - [Research] data explorer routing (3)
	    - Between views Where does this render to
	    - From saved object management 
	    - from dashboard 
	    - from legacy routes
    - [Research] feedback options (2)
    - Build basic doc table (5)
2.  Week 3 - 4 (June)
    - Build switching functionality between 'discover' and 'discover_legacy'. (?)
    - Build basic doc table (5)
    - Design state management & implement it (3)
    - Implement top nav and Querybar (3)
    - Implement basic view registry (1)
3.  Week 1 - 2 (July)
    - Build Sidepanel (8)
    - Build Discover 2.0 Data Visualization (3)

#### 2.10: Discover 2.0 feature complete

1.  Week 3 - 4 (July) - Week 1 - 2 (August)
    - Migrate single doc viewer (8)
    - Migrate Surrounding doc viewer (8)
    - Implement Saved object viewer (5)
    - Implement Embeddable (8)
1.  Week 3 - 4 (August)
    - Migrate all functional tests to FTR (8)
    - Perform comprehensive testing of 'discover' and 'data_explorer' plugins. (13)
    - Fix all discovered bugs and issues. (8)

#### 2.11: Discover legacy will be removed & VisBuilder will be integrated
1.  Week 1 - 2 (September)
    - Remove `discover_legacy`
    - Start `VisBuilder` to `data_explorer` migration
2.  Week 3 - 4 (September) - Week 1 - 2 (October)
    - Migrate Functional tests for VisBuilder 
    - Complete `VisBuilder` to `data_explorer` migration
    - Documentation of the Data Explorer, VisBuilder and Updated Discover plugins 
    - Blog post for Data Explorer

**Comment by DarshitChanpura** [comment]
@mnkugler Should this issue be moved to 2.11?

**Comment by mnkugler** [comment]
@DarshitChanpura Yes it should. Moved it to 2.11. Thanks!

**Comment by hdhalter** [comment]
Hi @ashwin-pc , will any of these items need documentation or updates to screenshots in the documentation? Thanks.

**Comment by ashwin-pc** [comment]
@hdhalter yeah there will be some updates needed to the screenshots for this change. The changes that have gone in until 2.11 are already reflected in the doc site. We will similarly update the screenshots for 2.12. @ananzh you are tracking this right?

**Comment by hdhalter** [comment]
> [@hdhalter](https://github.com/hdhalter) yeah there will be some updates needed to the screenshots for this change. The changes that have gone in until 2.11 are already reflected in the doc site. We will similarly update the screenshots for 2.12. [@ananzh](https://github.com/ananzh) you are tracking this right?

Thanks, @ashwin-pc ! Can someone create a doc issue describing what is changing for 2.13? Thanks!

**Comment by ashwin-pc** [comment]
@hdhalter There are no visible UI changes for 2.13
